folium/folium.py

```diff
@@ -218,15 +220,36 @@ def add_tile_layer(self, tile_name=None, tile_url=None, active=False):
             name of the tile layer
         tile_url: string
             url location of the tile layer
-        active: boolean
-            should the layer be active when added
+
+        For the available options see:
+            http://leafletjs.com/reference.html#tilelayer
+
         """
+        # Same defaults.
+        tms = kw.pop('tms', 'false')
+        minZoom = kw.pop('minZoom', 0)
+        opacity = kw.pop('opacity', 1)
+        maxZoom = kw.pop('maxZoom', 18)
+        noWrap = kw.pop('noWrap', 'false')
+        zoomOffset = kw.pop('zoomOffset', 0)
+        zoomReverse = kw.pop('zoomReverse', 'false')
+        continuousWorld = kw.pop('continuousWorld', 'false')
+
```

Comment:
Provided that you handle when `tms`,`noWrap`,`zoomReverse`,`continuousWorld` are `bool`, I would set defaults to `False` instead of `'false'`. What do you think ?
Anyway, this is a matter of taste and obviously not blocking for merge.